apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -366,12 +369,41 @@ export default class CoreLibrary {
     return speechBubbles[speechBubbles.length - 1];
   }
 
+  reachedSpriteMax() {
+    const numSprites = this.getNumberOfSprites();
+    return numSprites >= MAX_NUM_SPRITES;
```

Comment:
nit: you could probably just inline this? `return this.getNumberOfSprites() >= MAX_NUM_SPRITES`

Reply:
Thanks! Made the update.